src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php

```diff
@@ -29,11 +29,14 @@
 		TJDashboardUI.initDashboard(<?php echo ($this->item->state == 1? $this->item->dashboard_id : 0); ?>);
 	});
 </script>
-
-<div class="row-fluid">
-	<div class="col-xs-12 tjdashboard" style="margin-left:10px;">
-		<h3>
-			<div data-dashboard-id="<?php echo $this->item->dashboard_id;?>" class="tjdashboard-title"><?php echo htmlspecialchars($this->item->title);?></div>
-		</h3>
+<div class="<?php echo COM_TJDASHBOARD_WRAPPER_DIV; ?> tjBs3">
```

Comment:
If you are loading a constant for `COM_TJDASHBOARD_WRAPPER_DIV`, make `tjBs3` as part of wrapper class
eg. `COM_TJDASHBOARD_WRAPPER_DIV='class1 class2 tjBs3"`

Reply:
changed